carbon-design-system/carbon
# chore: add "all-contributors generate" to lint-staged actions

Ref #13466

To ensure the README is always reflecting the latest state of `.all-contributorsrc`, this PR adds a new pre-commit action that executes `all-contributors generate` whenever the `README.md` or `.all-contributorsrc` file is being updated.

#### Changelog

**New**

- New action for "lint-staged" pre-commit hook: `all-contributors generate`
  - Runs when `README.md` or `.all-contributorsrc` is updated

#### Testing / Reviewing

1. Pull this PR
2. Locally, change the `README.md` by updating general text and the "Contributors" table
3. Create a commit
4. The text change should remain in the commit while the table should've been re-generated based on `.all-contributorsrc`
